config/locales/provider_interface/make_offer.yml

```diff
@@ -49,6 +49,10 @@ en:
         success: Offer sent
       update:
         success: New offer sent
+    courses:
+      providers:
+        edit:
+          title: Training provider
```

Comment:
Consider introducing a new locale file as having the content stored in here will make it quite hard to find in the future. Maybe course.yml or change_course.yml

Reply:
👌